README.md

```diff
@@ -25,6 +25,10 @@ Diff JSON and JSON-like structures in Python.
 >>> diff({'a': [0, {'b': 4}, 1]}, {'a': [0, {'b': 5}, 1]})
 {'a': {1: {'b': 5}}}
 
+# You can exclude some jsonpaths from the diff (doesn't work if the value types are different)
+>>> diff({'a': 1, 'b': {'b1': 2, 'b2': 22}, 'c': 3},  {'a': 1, 'b': {'b1': 20, 'b2': 22}, 'c': 30}, exclude_paths=['b.b1', 'c'])
```

Comment:
Checking my understanding here:

The path should be an absolute path from the root of the object, right? if `b.b1` happens to be elsewhere in the object it is not intended to be excluded?